tasks/pipeline.py

```diff
@@ -810,6 +810,24 @@ def update_circleci_config(file_path, image_tag, test_version):
         circle.write(circle_ci.replace(f"{match.group(0)}", f"{image}:{image_tag}\n"))
 
 
+@task(
+    help={
+        "file_path": "path of the CircleCI configuration YAML file",
+    }
+)
+def get_image_tag(_, file_path=".circleci/config.yml"):
+    """
+    Print the current image tag of the given CircleCI configuration file (default: ".circleci/config.yml")
+    """
+    image_name = "gcr.io/datadoghq/agent-circleci-runner"
+    with open(file_path) as circle:
+        circle_ci = circle.read()
+    match = re.search(rf"({image_name}(_test_only)?):([a-zA-Z0-9_-]+)\n", circle_ci)
+    if not match:
+        raise RuntimeError(f"Impossible to find the version of image {image_name} in circleci configuration file")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏A move invoke-way of doing this would be to use the `Exit` class with a color message (for a better visibility in the logs). Feel free to ignore if you want a `RuntimeError` instead !
```
from invoke.exceptions import Exit

from tasks.libs.common.color import Color, color_message
```
```suggestion
        raise Exit(color_message(f"Impossible to find the version of image {image_name} in circleci configuration file", Color.RED), code=1)
```